events/using/aem-event-setup.md

```diff
@@ -26,14 +26,19 @@ Before setting up and using AEM with Adobe I/O, you will need to do the followin
 
 To complete this solution, you will need authorization to use the following services:
 
-*   An AEM instance, version 6.2 or 6.3, with administrative permissions. (**Note:** AEM screens in this topic are captured from version 6.3.)
+*   An AEM instance, version `6.2.x`, `6.3.x` or `6.4.x`, with administrative permissions. (**Note:** AEM screens in this topic are captured from version 6.3.)
 *   [Adobe I/O Console](https://adobe.io/console) access, with administrative permissions for your enterprise organization. If you do not have administrative permissions, please contact [ioevents-beta@adobe.com](mailto:ioevents-beta@adobe.com). After requesting administrative permissions, watch for an email from Adobe Systems Incorporated, notifying you of your permissions:
```

Comment:
I'm pretty sure we want to change this. We are no longer in beta, *and* we are not the ones in charge of giving admin access to Console. @cxs504 how do you suggest we rephrase this section?

Reply:
I removed the beta paragraph...

Reply:
How about
"If you do not have administrative permissions, please contact your marketing cloud administrator. After requesting administrative permissions..."